### PR TITLE
Use httpclient for graphql

### PR DESCRIPTION
**Pull request by duanxianbo**

### CODE REVIEW

File: src/client/invitation.client.ts [hunk 1]
```diff
@@ -93,7 +93,8 @@ export class InvitationClient {
     ): Promise<RemoteInvitation> {
         return this.graphQLClient
             .mutate({
-                mutation: gql`
+                operationName: "createInvitation",
+                query: `
```
Comment (naturetime10) [inline src/client/invitation.client.ts]:
```suggestion
                mutation: `
```

File: src/client/invitation.client.ts [hunk 2]
```diff
@@ -21,7 +19,8 @@ export class InvitationClient {
     ): Promise<RemoteInvitation> {
         return this.graphQLClient
             .query({
-                query: gql`
+                operationName: "getInvitations",
```
Comment (naturetime10) [inline src/client/invitation.client.ts]: Can this be optional?

File: src/storage/syncer/stateMutators.ts [hunk 3]
```diff
@@ -388,6 +388,7 @@ export function mergeTask(
     currAppState: AppState,
     remoteTask: RemoteTask,
 ): AppState {
+    debugger
```
Comment (naturetime10) [inline src/storage/syncer/stateMutators.ts]:
```suggestion
```

**Review by naturetime10 — CHANGES_REQUESTED** [review]

File: src/client/invitation.client.ts (hunk 2 above)
Comment (duanxianbo) [reply src/client/invitation.client.ts]: yep

**Review by naturetime10 — APPROVED** [review]

**Pull request merged**